Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListValidationFailureRepository.cs

```diff
@@ -102,6 +104,8 @@ public void AddValidationFailuresForBocList (IEnumerable<BusinessObjectValidatio
 
       _listValidationFailures.AddRange(
           validationFailures.Select(e => new HandleableFailure(BocListValidationFailureWithLocationInformation.CreateFailure(e))));
+
+      ValidationFailureAdded?.Invoke();
```

Comment:
please extract into dedicated `OnValidationFailureAdded` method. it's one of those standard patterns for events in .net